docs/cql/tombstone-gc.rst

```diff
@@ -0,0 +1,109 @@
+=====================================
+Tombstone Garbage Collection (GC) 
+=====================================
+
+Overview
+----------
+When you delete some data from the database, the data record is not immediately 
+removed. Instead, it is marked with a tombstone - a data marker that prevents 
+the data from being returned as a query result. The data covered by the tombstone 
+and the tombstone itself are eventually removed after the compaction process 
+is performed.
+
+ScyllaDB allows you to manage how and when tombstones are removed, considering 
+that tombstones must be garbage-collected:
+
+* Within a reasonable time because a large number of tombstones impacts query 
+  response times and leads to performance degradation.
+* After a cluster-wide repair to ensure that all replicas are in sync. Otherwise, 
+  tombstones might be missing on some replica nodes, which may lead to 
+  the resurrection of deleted data.
```

Comment:
The last paragraph is ambiguous. You're saying that tombstones "must be garbage collected after a cluster-wide repair". It might sound like you're saying they must be garbage collected after each repair, but that's not the case - the point is that you're not allowed to do gc **before** the repair, not that you need to do gc **after** each gc.